src/reporters/tiny.js

```diff
@@ -0,0 +1,66 @@
+import path from "path";
+
+export default report;
+
+async function report(runningTests, startTime) {
+  console.log("");
+  const summary = await reportSequentially(runningTests);
+  const { failed, passed, endTime } = summary;
+  const ok = color.green(`✓ ${passed}`);
+  const notOk = failed ? color.red(`✗ ${failed}`) : "";
+  const runTime = color.grey(`Ran in ${endTime - startTime}ms`);
+  console.log(`${ok} ${notOk} ${runTime}`);
+}
+
+async function reportSequentially(runningTests) {
+  let failed = 0;
+  for await (const result of runningTests) {
+    if (result.error) {
+      failed++;
+      console.log(formatResult(result));
+    }
+  }
+  return { failed, passed: runningTests.length - failed, endTime: Date.now() };
+}
+
+function formatResult(result) {
+  const { error, title } = result;
+  const { message, stack, operator } = error;
+  const header = color.red(`✗ ${title}`);
+  const path = color.grey(getFilePath(stack));
+  const errorMsg = formatMessage(message, operator);
+  return `${header}\n${path}\n${errorMsg}\n`;
+}
+
+function getFilePath(stack) {
+  const absolute = stack.match(/^ *at.*\(file:\/\/(.*?)\)/m)[1];
+  return path.relative(process.cwd(), absolute);
+}
+
+function formatMessage(message, operator) {
+  if (operator === "deepStrictEqual") {
+    return message.split("\n").slice(3).join("\n");
+  }
+  if (operator === "strictEqual") {
+    const removeEmptyLines = (line) => line !== "";
```

Comment:
another way to do this is to use the `Boolean` constructor `.filter(Boolean)` will remove `''`'s

Reply:
I'm not entirely sure. Using `Boolean` reads as "keep truthy values", and going from there to the true intent may not be immediately obvious to everyone :/

I was going to go for something like `.replace(/\n\n/g, "\n")` which makes more sense to me and is shorter and a bit more performant (not that it matters in this case), but in the end I used a named filter function to clarify the intent. 

In any case I'm not very happy with the whole parsing the error message strings thingy, I'm going to look into alternative approaches :/